[Bug] Compilation error for new version of r.landscape.evol

**Name of the addon**
r.landscape.evol

**Describe the bug**
Hi all, I have just added a substantially new version of r.landscape.evol to the repo, but for some reason I cannot get it to install correctly with g.extension. I can't think of any code change that would do this, and I believe I have configured the Makefile and html documentation properly. Here's the error from g.extension, and any suggestions on what to change will be very much appreciated!

```
g.extension extension=r.landscape.evol                                          
Fetching <r.landscape.evol> from GRASS GIS Addons repository (be patient)...
Compiling...
Traceback (most recent call last):
  File "/usr/lib/grass78/tools/g.html2man.py", line 45, in
main
    p.feed(line)
  File "/usr/lib/python3.8/html/parser.py", line 111, in
feed
    self.goahead(0)
  File "/usr/lib/python3.8/html/parser.py", line 173, in
goahead
    k = self.parse_endtag(i)
  File "/usr/lib/python3.8/html/parser.py", line 421, in
parse_endtag
    self.handle_endtag(elem)
  File "/usr/lib/grass78/tools/ghtml.py", line 231, in
handle_endtag
    if self.pop() == tag:
  File "/usr/lib/grass78/tools/ghtml.py", line 185, in pop
    self.excluded = self.excluded_stack.pop()
IndexError: pop from empty list
During handling of the above exception, another exception
occurred:
Traceback (most recent call last):
  File "/usr/lib/grass78/tools/g.html2man.py", line 78, in
<module>
    main()
  File "/usr/lib/grass78/tools/g.html2man.py", line 49, in
main
    (infile, err.lineno, err.offset, err.msg))
AttributeError: 'IndexError' object has no attribute
'lineno'
make: *** [/usr/lib/grass78/include/Make/Html.make:11: /tmp/
grass7-iullah-439490/tmpnphof_e7/r.landscape.evol/docs/man/m
an1/r.landscape.evol.1] Error 1
ERROR: Compilation failed, sorry. Please check above error messages.
```

**To Reproduce**
Steps to reproduce the behavior:
1. Go to g.extension
2. Raster>  r.landscape.evol
3. Install
4. See error

**Expected behavior**
Should compile without error.


**System description (please complete the following information):**
 - Linux, Pop!_OS 20.04
 - GRASS GIS version [e.g. 7.8.2]

--- a/grass7/raster/r.landscape.evol/r.landscape.evol.html
+++ b/grass7/raster/r.landscape.evol/r.landscape.evol.html
@@ -540,78 +540,78 @@ Academic Press.</p>
 
 <p>Aiello, A., Adamo, M., Canora, F., 2015. Remote sensing and
 GIS to assess soil erosion with RUSLE3D and USPED at river
-basin scale in southern Italy. CATENA 131, 174–185. <a
+basin scale in southern Italy. CATENA 131, 174-185. <a
 href="https://doi.org/10.1016/j.catena.2015.04.003">https://doi.org/10.1016/j.catena.2015.04.003</a></p>
 
 <p>Aksoy, H., Kavvas, M.L., 2005. A review of hillslope and watershed
-scale erosion and sediment transport models. CATENA 64, 247–271. <a
+scale erosion and sediment transport models. CATENA 64, 247-271. <a
 href="https://doi.org/10.1016/j.catena.2005.08.008">https://doi.org/10.1016/j.catena.2005.08.008</a></p>
 
 <p>Ayala, G., French, C., 2005. Erosion modeling of past land-use practices in
 the Fiume di Sotto di Troina river valley, north-central Sicily. Geoarchaeology
-20, 149–167.</p>
+20, 149-167.</p>
 
 <p>Benavidez, R., Jackson, B., Maxwell, D., Norton, K., 2018. A
 review of the (Revised) Universal Soil Loss Equation (R/USLE): with
 a view to increasing its global applicability and improving soil loss
-estimates. Hydrology and Earth System Sciences Discussions 1–34. <a
+estimates. Hydrology and Earth System Sciences Discussions 1-34. <a
 href="https://doi.org/10.5194/hess-2018-68">https://doi.org/10.5194/hess-2018-68</a></p>
 
 <p>Bosco, C., de Rigo, D., Dewitte, O., Poesen, J., Panagos, P.,
 2015. Modelling soil erosion at European scale: towards harmonization and
-reproducibility. Natural Hazards and Earth System Science 15, 225–245. <a
+reproducibility. Natural Hazards and Earth System Science 15, 225-245. <a
 href="https://doi.org/10.5194/nhess-15-225-2015">https://doi.org/10.5194/nhess-15-225-2015</a></p>
 
 <p>Davy, P., Crave, A., 2000. Upscaling local-scale transport
 processes in large-scale relief dynamics. Physics and Chemistry
-of the Earth, Part A: Solid Earth and Geodesy 25, 533–541. <a
+of the Earth, Part A: Solid Earth and Geodesy 25, 533-541. <a
 href="https://doi.org/10.1016/S1464-1895(00)00082-X">https://doi.org/10.1016/S1464-1895(00)00082-X</a></p>
 
 <p>Dietrich, W.E., Bellugi, D.G., Sklar, L.S., Stock, J.D., Heimsath, A.M.,
 Roering, J.J., 2003. Geomorphic Transport Laws for Predicting Landscape
 form and Dynamics, in: Wilcock, P.R., Iverson, R.M. (Eds.), Prediction
 in Geomorphology, Geophysical Monograph. American Geophysical Union,
-pp. 103–132.</p>
+pp. 103-132.</p>
 
 <p>Diodato, N., 2006. Predicting RUSLE (Revised Universal Soil Loss
 Equation) Monthly Erosivity Index from Readily Available Rainfall
-Data in Mediterranean Area. The Environmentalist 26, 63–70. <a
+Data in Mediterranean Area. The Environmentalist 26, 63-70. <a
 href="https://doi.org/10.1007/s10669-006-5359-x">https://doi.org/10.1007/s10669-006-5359-x</a></p>
 
 <p>Hammad, A.A., Lundekvam, H., Børresen, T., 2004. Adaptation of RUSLE in
 the Eastern Part of the Mediterranean Region. Environmental Management 34,
-829–841.</p>
+829-841.</p>
 
 <p>Hancock, G.R., 2004. Modelling soil erosion on the catchment and landscape
-scale using landscape evolution models – a probabilistic approach using
+scale using landscape evolution models - a probabilistic approach using
 digital elevation model error, in: Super Soil 2004:3rd Australian New Zealand
 Soils Conference. University of Sydney, Australia.</p>
 
 <p>Kelley, A.D., Malin, M.C., Nielson, G.M., 1988. Terrain simulation using
-a model of stream erosion. ACM SIGGRAPH Computer Graphics 22, 263–268.</p>
+a model of stream erosion. ACM SIGGRAPH Computer Graphics 22, 263-268.</p>
 
 <p>Koko, Š., 2011. Simulation of gully erosion using the SIMWE model and
-GIS. Landform Analysis 17, 81–86.</p>
+GIS. Landform Analysis 17, 81-86.</p>
 
 <p>Kwang, J.S., Parker, G., 2017. Landscape evolution models using the stream
-power incision model show unrealistic behavior when &lt;i&gt;m&lt;/i&gt;
-∕ &lt;i&gt;n&lt;/i&gt; equals 0.5. Earth Surface Dynamics 5, 807–820. <a
+power incision model show unrealistic behavior when m / n equals 0.5.
+Earth Surface Dynamics 5, 807-820. <a
 href="https://doi.org/10.5194/esurf-5-807-2017">https://doi.org/10.5194/esurf-5-807-2017</a></p>
 
 <p>Martínez-Casasnovas, J.A., Sánchez-Bosch, I., 2000. Impact assessment
 of changes in land use/conservation practices on soil erosion in the
 Penedès-Anoia vineyard region (NE Spain). Soil and Tillage Research 57,
-101–106.</p>
+101-106.</p>
 
 <p>Mathier, L., Roy, A.G., Paré, J.P., 1989. The effect of
 slope gradient and length on the parameters of a sediment
-transport equation for sheetwash. CATENA 16, 545–558. <a
+transport equation for sheetwash. CATENA 16, 545-558. <a
 href="https://doi.org/10.1016/0341-8162(89)90041-6">https://doi.org/10.1016/0341-8162(89)90041-6</a></p>
 
 <p>Mitasova, H., Barton, C.M., Ullah, I.I., Hofierka, J., Harmon,
 R.S., 2013. GIS-based soil erosion modeling, in: Shroder, J., Bishop,
 M.P. (Eds.), Remote Sensing and GIScience in Geomorphology, Treatise in
-Geomorphology. Academic Press, San Diego, pp. 228–258.</p>
+Geomorphology. Academic Press, San Diego, pp. 228-258.</p>
 
 <p>Mitasova, H., Brown, W.M., Johnston, D., 2002. Terrain Modeling and Soil
 Erosion Simulation Final Report. Geographic Modeling Systems Lab, University
@@ -619,7 +619,7 @@ of Illinois at Urbana-Champaign.</p>
 
 <p>Mitasova, H., Hofierka, J., Zlocha, M., Iverson, L.R., 1996a. Modelling
 topographic potential for erosion and deposition using GIS. International
-journal of geographical information systems 10, 629–641. <a
+journal of geographical information systems 10, 629-641. <a
 href="https://doi.org/10.1080/02693799608902101">https://doi.org/10.1080/02693799608902101</a></p>
 
 <p>Mitasova, H., Mitas, L., Brown, W.M., 2001. Multiscale Simulation
@@ -627,7 +627,7 @@ of Land Use Impact on Soil Erosion and Deposition Patterns, in: Stott,
 D.E., Mohtar, R.H., Steinhardt, G.C. (Eds.), Sustaining the Global Farm:
 10th International Soil Conservation Organization Meeting Held May 24-29,
 1999. Purdue University and the USDA-ARS National Soil Erosion Research
-Laboratory, pp. 1163–1169.</p>
+Laboratory, pp. 1163-1169.</p>
 
 <p>Mitasova, H., Mitas, L., Brown, W.M., Johnston, D., 1996b. Multidimensional
 Soil Erosion/Deposition Modeling Part III: Process based erosion
@@ -642,73 +642,73 @@ at Urbana-Champaign.</p>
 <p>Onori, F., De Bonis, P., Grauso, S., 2006. Soil erosion prediction at
 the basin scale using the revised universal soil loss equation (RUSLE)
 in a catchment of Sicily (southern Italy). Environmental Geology 50,
-1129–1140.</p>
+1129-1140.</p>
 
 <p>Panagos, P., Ballabio, C., Borrelli, P., Meusburger, K., Klik,
-A., Rousseva, S., Tadić, M.P., Michaelides, S., Hrabalíková,
+A., Rousseva, S., Tadi&#263;, M.P., Michaelides, S., Hrabal&iacute;kov&aacute;,
 M., Olsen, P., Aalto, J., Lakatos, M., Rymszewicz, A., Dumitrescu,
 A., Beguería, S., Alewell, C., 2015a. Rainfall erosivity in
-Europe. Science of The Total Environment 511, 801–814. <a
+Europe. Science of The Total Environment 511, 801-814. <a
 href="https://doi.org/10.1016/j.scitotenv.2015.01.008">https://doi.org/10.1016/j.scitotenv.2015.01.008</a></p>
 
 <p>Panagos, P., Borrelli, P., Meusburger, K., Alewell, C., Lugato, E.,
 Montanarella, L., 2015b. Estimating the soil erosion cover-management
-factor at the European scale. Land Use Policy 48, 38–50. <a
+factor at the European scale. Land Use Policy 48, 38-50. <a
 href="https://doi.org/10.1016/j.landusepol.2015.05.021">https://doi.org/10.1016/j.landusepol.2015.05.021</a></p>
 
 <p>Panagos, P., Borrelli, P., Meusburger, K., van der Zanden, E.H.,
 Poesen, J., Alewell, C., 2015c. Modelling the effect of support
 practices (P-factor) on the reduction of soil erosion by water at
-European scale. Environmental Science &amp; Policy 51, 23–34. <a
+European scale. Environmental Science &amp; Policy 51, 23-34. <a
 href="https://doi.org/10.1016/j.envsci.2015.03.012">https://doi.org/10.1016/j.envsci.2015.03.012</a></p>
 
 <p>Panagos, P., Meusburger, K., Ballabio, C., Borrelli, P., Alewell,
 C., 2014. Soil erodibility in Europe: A high-resolution dataset based
-on LUCAS. Science of The Total Environment 479–480, 189–200. <a
+on LUCAS. Science of The Total Environment 479-480, 189-200. <a
 href="https://doi.org/10.1016/j.scitotenv.2014.02.010">https://doi.org/10.1016/j.scitotenv.2014.02.010</a></p>
 
 <p>Peckham, S.D., 2003. Fluvial landscape models and catchment-scale
-sediment transport. Global and Planetary Change 39, 31–51. <a
+sediment transport. Global and Planetary Change 39, 31-51. <a
 href="https://doi.org/10.1016/S0921-8181(03)00014-6">https://doi.org/10.1016/S0921-8181(03)00014-6</a></p>
 
 <p>Peeters, I., Rommens, T., Verstraeten, G., Govers, G., Van Rompaey, A.,
 Poesen, J., Van Oost, K., 2006. Reconstructing ancient topography through
-erosion modelling. Geomorphology 78, 250–264.  Pistocchi, A., Cassani, G.,
+erosion modelling. Geomorphology 78, 250-264.  Pistocchi, A., Cassani, G.,
 Zani, O., n.d. Use of the USPED model for mapping soil erosion and managing
 best land conservation practices 7.</p>
 
 <p>Renard, K.G., Foster, G.R., Weesies, G.A., McCool, D.K., Yoder, D.C.,
 1997. Predicting soil erosion by water: a guide to conservation planning
 with the Revised Universal Soil Loss Equation (RUSLE), in: Agriculture
-Handbook. US Department of Agriculture, Washington, DC, pp. 1–251.</p>
+Handbook. US Department of Agriculture, Washington, DC, pp. 1-251.</p>
 
 <p>Renard, K.G., Foster, G.R., Weesies, G.A., Porter, J.P., 1991. RUSLE:
 Revised Universal Soil Loss Equation. Journal of Soil and Water Conservation
-46, 30–33.</p>
+46, 30-33.</p>
 
 <p>Renard, K.G., Freimund, J.R., 1994. Using monthly precipitation data
 to estimate the R-factor in the revised USLE. Journal of Hydrology 157,
-287–306.</p>
+287-306.</p>
 
 <p>Sklar, L.S., Riebe, C.S., Marshall, J.A., Genetti, J., Leclere, S., Lukens,
 C.L., Merces, V., 2017. The problem of predicting the size distribution of
-sediment supplied by hillslopes to rivers. Geomorphology 277, 31–49. <a
+sediment supplied by hillslopes to rivers. Geomorphology 277, 31-49. <a
 href="https://doi.org/10.1016/j.geomorph.2016.05.005">https://doi.org/10.1016/j.geomorph.2016.05.005</a></p>
 
 <p>Terranova, O., Antronico, L., Coscarelli, R., Iaquinta,
 P., 2009. Soil erosion risk scenarios in the Mediterranean
 environment using RUSLE and GIS: An application model for
-Calabria (southern Italy). Geomorphology 112, 228–245. <a
+Calabria (southern Italy). Geomorphology 112, 228-245. <a
 href="https://doi.org/10.1016/j.geomorph.2009.06.009">https://doi.org/10.1016/j.geomorph.2009.06.009</a></p>
 
 <p>Tucker, G.E., Whipple, K.X., 2002. Topographic outcomes predicted by stream
 erosion models: Sensitivity analysis and intermodel comparison. J. Geophys. Res
-107, 1–1.</p>
+107, 1-1.</p>
 
 <p>Warren, S.D., Mitasova, H., Hohmann, M.G., Landsberger, S., Skander, F.Y.,
 Ruzycki, T.S., Senseman, G.M., 2005. Validation of a 3-D enhancement of the
 Universal Soil Loss Equation for preediction of soil erosion and sediment
-deposition. Catena 64, 281–296.</p>
+deposition. Catena 64, 281-296.</p>
 
 <p>Whipple, K.X., Tucker, G.E., 2002. Implications of sediment-flux-dependent
 river incision models for landscape evolution. Journal of Geophysical
@@ -720,7 +720,7 @@ response timescales, and research needs. Journal of Geophysical Research
 104, 17,661-17,674.</p>
 
 <p>Willgoose, G., 2005. Mathematical Modeling of Whole Landscape
-Evolution. Annual Review of Earth and Planetary Sciences 33, 443–459.</p>
+Evolution. Annual Review of Earth and Planetary Sciences 33, 443-459.</p>
 
 <h2>AUTHORS</h2>
 

